Makefile

```
@@ -91,6 +93,11 @@ MSVCC64 = cl.exe $(MSVC_FLAGS)
 endif
 endif
 
+ifeq ($(shell expr "$(CI)" : '^[Tt]rue$$'),4)
```

I think this avoids the shell call (which matters a lot to me... cf. #119)
```suggestion
ifneq "$(filter True true,$(CI))" ""
```